### PR TITLE
add support for array sortBy

### LINKED ISSUE #447: add array `sortBy`

### PR DESCRIPTION
**Pull request by mum-never-proud**
Resolves #447

### CODE REVIEW

File: addon/array/index.js [hunk 1]
```diff
@@ -28,3 +28,4 @@ export { default as sort } from './sort';
 export { default as uniqBy } from './uniq-by';
 export { default as uniq } from './uniq';
 export { default as without } from './without';
+export { default as sortBy } from './sort-by';
```
Comment (kellyselden) [inline addon/array/index.js:31]: can you order this by sort?

**Comment by mum-never-proud** [comment]
I made the changes, but I really doubt the word choice in readme.md, let me know your thoughts on this

cc @kellyselden

File: README.md [hunk 2]
```diff
@@ -562,6 +563,14 @@ value1: array.slice('array', 1), // [2, 3]
 value2: array.slice('array', difference('array.length', 1)) // [3]
 ```
 
+##### `array.sortBy`
+sorts the given array of objects by key
+```js
+array1: Ember.A([{ key: 'abc' }, { key: 'xyz' }]),
+value1: array.sort('array1', 'key'), // [{ key: 'abc' }, { key: 'xyz' }]
```
Comment (kellyselden) [inline README.md:570]: should be sortBy

Comment (mum-never-proud) [reply README.md]: sorry my bad

**Pull request merged**